packages/release/mnt.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Mnt Directory (/mnt)
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
```

Comment:
Why is `umount.target` included in the `Before` line?

Reply:
That's how we mount other directories like `/var` and `/opt`. According to the systemd's documentation:

> All mount units acquire automatic Before= and Conflicts= on umount.target in order to be stopped during shutdown.

So I guess those two lines aren't actually needed, and I think we added them to clarify what dependencies the unit has.